web/src/compose_validate.ts

```diff
@@ -544,6 +544,10 @@ export function validate_stream_message_mentions(opts: StreamWildcardOptions): b
 }
 
 export function validate_stream_message_address_info(sub: StreamSubscription): boolean {
+    if (sub.is_archived) {
+        compose_banner.show_stream_does_not_exist_error(sub.name);
+        return false;
+    }
```

Comment:
Should we send some specific banner for archived stream instead of stream does not exist?

While it is true that we are not sending archived streams back to client in this PR,  but the only time this code will be executed is when a stream is archived and user has the stream open and does not refresh.

So it makes sense to have an "This stream has been archived banner"? Or should we have this banner in #28478?

Reply:
Yeah, I think we should have a banner, "This channel has been archived" seems OK for first-pass text.